### DISCUSSION TITLE
Links missing in markdown files when importing to documents

### DISCUSSION THREAD

**Discussion by hame25** [open-webui/open-webui#2172] [General]
# Bug Report

## Description

**Bug Summary:**
Importing a markdown file in to documents does not preserve href's

When asking for a url to be shared this is not available

**Steps to Reproduce:**
Create a markdown file with a link in it

e.g. `[Access my custom docs here]("http://www.my-docs-url.com")`

import it into documents

run a question on given documents

"Can you share url for "my custom docs""

Does not return the link

**Expected Behavior:**
Link is returned

**Actual Behavior:**
"My custom docs can be found at: [Insert URL]"

## Environment

- **Open WebUI Version:** 0.1.124
- **Ollama (if applicable):** 0.1.34

- **Operating System:** macOS sonoma 14.4.1 
- **Browser (if applicable):** Chrome 123

## Reproduction Details

**Confirmation:**

- [X ] I have read and followed all the instructions provided in the README.md.
- [X ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**

**Docker Container Logs:**

**Screenshots (if applicable):**

## Installation Method

Docker quickstart described here: https://docs.openwebui.com/getting-started/#quick-start-with-docker-

**Comment by NL647** [comment]
Hi, I have same issue.
Did you find a solution?
